Svg.Editor.Samples.Forms/BindableToolbarProperty.cs

```diff
@@ -15,45 +11,50 @@ public static class BindableToolbarProperty
         private static void ToolbarItemsChanged(BindableObject bindable, object oldValue, object newValue)
         {
             var page = bindable as Page;
-
             if (page == null) return;
 
-            if (oldValue != null)
+            var otbi = page.ToolbarItems;
```

Comment:
page.ToolbarItems are now an ObservableCollection. [See here](https://github.com/dotnet/maui/blob/71e6bb5610dfd000cf63f48f03b259ae8dce51cd/src/Controls/src/Core/Page/Page.cs#L65)
So just replacing all contents is too slow as every change triggers a UI refresh.